src/adt/siphash.c

```diff
@@ -19,6 +19,9 @@
 #include <stdio.h>
 #include <string.h>
 
+/* Added to suppress UBSan warning for (expected) unsigned integer overflow. */
```

Comment:
(then there's no need for this comment, given the purpose of the header)

Reply:
Okay. I was just being extra-explicit because it's modifying a reference implementation.